lms/static/sass/shared/_footer-edx.scss

```diff
@@ -117,17 +218,38 @@ footer#footer-edx-v3 {
     color: $edx-footer-link-color;
   }
 
+
+
   a.sm-link {
     @include float(left);
-    @include margin(0, 0, 10px, 10px);
-    @include font-size(28);
+    @include margin(0, 6px, 10px, 0px);
+    @include font-size(40);
     @include line-height(28);
     width: 35px;
     height: 30px;
     line-height: 1;
     position: relative;
     display: inline;
     background: none;
+    @include text-align(left);
+    &:first-of-type {
+      @include margin-left(0);
+    }
+
+    &:hover,
+    &:focus {
+      opacity: 0.7;
+      border: none;
+    }
+  }
+    a.social-links {
+    @include margin(0, -7px, 10px, 0px);
+    @include font-size(45);
```

Comment:
This is surprising. Is it meant to be 45?

Reply:
yes the social app links icons are of very small size. Maybe there is something to do with the icons.